Makefile

```diff
@@ -13,6 +13,40 @@ $(warning Undefining GOFLAGS set in CI)
 undefine GOFLAGS
 endif
 
+# Tools #
+
+# The tools required to build and test the project come from two sources:
+# 1. .bingo/Variables.mk: tools that are orthogonal to OLM, e.g.
+#   - golangci-lint
+#   - helm
+#   - kind
+#   - setup-envtest
+#   - yq
+# 2. tools.go: imports modules that need to be vendored and/or have a low tolerance for skew with the main module
+#              or because they are already imported by the main module, but we want to pull down the whole dependency
+#              to run some cmd or script, or get some resource (e.g. CRD yaml files).
+#   - OLM API CRDs
+#   - Test libraries, e.g. ginkgo
```

Comment:
As far as I see we do not have ginkgo in `tools.go` - only here in `Makefile`.

Any reason why not to have it managed by bingo?

Reply:
I forgot I removed it to see if it was necessary XD
I was thinking it would be good to execute the tests with the same library version as what is in go.mod. That's why I decided to not bingo it. I'd like to have a few things as possible in bingo because dependabot doesn't track those for updates =P (although I'm sure we could come up with a bot of our own)

Reply:
@perdasilva I see the point about dependabot. But right now we do `go run` in `Makefile` - I think it will have the same effect since on dependabot since ginkgo won't end up in go.mod. Or am I missing something?

Reply:
ginkgo is in go.mod anyway because we import it in the e2e and for some unit tests - so I think dependabot should pick it up right? So, we could have skew between bingo gingko and olm ginkgo